packages/replayio/src/utils/initialization/initialize.ts

```diff
@@ -60,10 +60,7 @@ export async function initialize({
     : Promise.resolve();
 
   const mixpanelPromise = raceWithTimeout(
-    initMixpanelForUserSession(accessToken, {
-      packageName,
-      packageVersion,
-    }),
+    mixpanelAPI.initialize({ accessToken, packageName, packageVersion }),
```

Comment:
not directly related to this PR - but since the logger initialization is grouped together with the mixpanel's initialization in our test plugins it could also make sense to move that into this `[...]/initialize.ts` file here